### PR TITLE
[py2/py3] dict.iteritems() --> six.iteritems(dict)

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: examples/test_examples.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from __future__ import print_function, division, absolute_import
+import six
```
Comment (campagnola) [inline examples/test_examples.py]: I'd rather not add a dependency for this.. we can embed `six.py` under `pyqtgraph/util`, though.
Presently I use `pyqtgraph/python2_3.py` for this purpose (as it predates my awareness of `six`).

**Comment by lidstrom83** [comment]
Two things that jumped out:
1. Needs an import of six in pyqtgraph/multiprocess/processes.py
2. Needs the six dependency removed from install_requires in setup.py

**Comment by ericdill** [comment]
@lidstrom83 good catches, thanks!

@campagnola bump, this is probably ready to go, have another look?

**Comment by lesauxvi** [comment]
ping @campagnola . Ready to go?

**Pull request closed without merging**